examples/nestedmodel/src/main/kotlin/io/fritz2/examples/nestedmodel/nestedmodel.kt

```diff
@@ -0,0 +1,75 @@
+package io.fritz2.examples.nestedmodel
+
+import io.fritz2.binding.Lens
+import io.fritz2.binding.Store
+import io.fritz2.binding.each
+import io.fritz2.dom.html.html
+import io.fritz2.dom.mount
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.FlowPreview
+import kotlinx.coroutines.flow.map
+import io.fritz2.binding.map
+
+
+data class Outer(val inner: Inner, val value: String) {
+    companion object {
+        val innerLens = object : Lens<Outer, Inner> {
+            override fun get(parent: Outer): Inner = parent.inner
+            override fun set(parent: Outer, value: Inner): Outer = parent.copy(inner = value)
+        }
+
+        val valueLens = object : Lens<Outer, String> {
+            override fun get(parent: Outer): String = parent.value
+            override fun set(parent: Outer, value: String): Outer = parent.copy(value = value)
+        }
+    }
+}
+
+data class Inner(val value: String) {
+    companion object {
+        val valueLens = object : Lens<Inner, String> {
+            override fun get(parent: Inner): String = parent.value
+            override fun set(parent: Inner, value: String): Inner = parent.copy(value = value)
+        }
+    }
+}
```

Comment:
Does it work with seqs when other pull-request is completed?

Reply:
not yet. working on it.